scripts/exregional_run_analysis.sh

```diff
@@ -302,20 +304,20 @@ fi
 #-----------------------------------------------------------------------
 
 FV3SARPATH=${CYCLE_DIR}
-cp_vrfy ${fixdir}/fv3_akbk                               fv3_akbk
-cp_vrfy ${fixdir}/fv3_grid_spec                          fv3_grid_spec
+cp_vrfy ${fixgriddir}/fv3_akbk                               fv3_akbk
+cp_vrfy ${fixgriddir}/fv3_grid_spec                          fv3_grid_spec
 
-if [ ${BKTYPE} -eq 1 ]; then  # Use background from INPUT
+if [ ${BKTYPE} -eq 1 ]; then  # cold start uses background from INPUT
   cp_vrfy ${bkpath}/gfs_data.tile7.halo0.nc                gfs_data.tile7.halo0.nc_b
-  ${NCKS} -A -v  phis ${fixdir}/phis.nc                    gfs_data.tile7.halo0.nc_b
-  ${NCKS} -A -v radar_tten ${fixdir}/radar_tten_input.nc   gfs_data.tile7.halo0.nc_b
+  ${NCKS} -A -v  phis ${fixgriddir}/phis.nc                    gfs_data.tile7.halo0.nc_b
+  ${NCKS} -A -v radar_tten ${fixgriddir}/radar_tten_input.nc   gfs_data.tile7.halo0.nc_b
```

Comment:
A comment here to describe what the NCKS command is doing and why would be helpful.

Reply:
Done

Reply:
I will remove the radar_tten one when I get a chance to work on the radar then code. The first one needs to work with GSI and EMC but should not be a big effort.